ThunderDome/ThunderDome/monsterBlock.cs

```diff
@@ -4,6 +4,79 @@
 
 namespace ThunderDome
 {
+    class arena {
```

Comment:
created arena to make the class objects "cleaner".   The arena stuff is not a part of a monster.  Think about if you want to do something like run many fights simultaneously.  To do so you would  adjust the area the monster should not have to change.

Also, doing it this way lets you avoid the use of `static` methods.   Static methods are a special case so if you are using them alot then you might be doing something wrong.